apps/daffio/src/app/docs/design/pages/component/component.html

```diff
@@ -0,0 +1,29 @@
+<ng-container *ngrxLet="doc$; let doc">
+	<daffio-doc-viewer [doc]="doc.guide">
+		<daff-tabs>
+			<daff-tab>
+				<daff-tab-label>
+					Usage
+				</daff-tab-label>
+				<daff-tab-panel>
+					<daff-article
+						[innerHtml]="getInnerHtml(doc.guide)">
+					</daff-article>
+				</daff-tab-panel>
+			</daff-tab>
+
+			<daff-tab>
+				<daff-tab-label>
+					API
+				</daff-tab-label>
+				<daff-tab-panel>
+					@for (apiDoc of doc.api; track $index) {
+						<daff-article
```

Comment:
Some elements inside here isn't styled by the `DaffArticleComponent`.

i.e. the `DAFF_ACCORDION_COMPONENTS` link doesn't have the article a tag styles, but the table does.

<img width="1059" alt="Screen Shot 2024-12-30 at 12 21 02 PM" src="https://github.com/user-attachments/assets/05551c56-b798-4bb3-93c8-d6735a46e300" />
<img width="1074" alt="Screen Shot 2024-12-30 at 12 20 54 PM" src="https://github.com/user-attachments/assets/0b547c39-beb0-4c89-951f-ca1d25b66524" />

Reply:
odd...do you know how to fix this?

Reply:
It seems like it's only happening on design docs which is weird because packages and api also use the same `daffio-doc-viewer` component right?

I've been getting this error when I serve daffio, could this be related?

```
ERROR Error: ENOENT: no such file or directory, open '/Users/elain/Desktop/Graycore/dist/apps/daffio/browser/assets/daffio/docs/design/index.json'
    at async open (node:internal/fs/promises:639:25)
    at async Module.readFile (node:internal/fs/promises:1242:14) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/Users/elain/Desktop/Graycore/dist/apps/daffio/browser/assets/daffio/docs/design/index.json'
}
```

Reply:
Specifically on design components pages. Article styles are being rendered correctly on pages like getting started and all the foundation pages.

Reply:
> I've been getting this error when I serve daffio, could this be related?

I'm not seeing that, make sure you are building everything (dgeni plus daffio build so all files are copied)

Reply:
> Specifically on design component pages. Article styles are being rendered correctly on pages like getting started and all the foundation pages.

What's odd is that the code article styles are being applied, just not the anchor ones.

Reply:
Heading styles are also not applied.

Reply:
> > I've been getting this error when I serve daffio, could this be related?
> 
> I'm not seeing that, make sure you are building everything (dgeni plus daffio build so all files are copied)

I have everything built. I even deleted dist and node_modules and rebuilt everything to check. The error goes away when I save a file, but reappears if I hit refresh.

Reply:
I figured out why the styles aren't applying. I have `DaffArticleEncapsulatedDirective` as a hostDirective in the tabs component. I'll remove it in a separate PR.

Reply:
fixed in https://github.com/graycoreio/daffodil/pull/3387